src/serac/physics/solid.hpp

```diff
@@ -311,7 +317,7 @@ class Solid : public BasePhysics {
   /**
    * @brief Destroy the Nonlinear Solid Solver object
    */
-  virtual ~Solid() = default;
```

Comment:
I must have jinxed it with my comment yesterday 😆

Reply:
I blame @jonwong12 for finding this problem!